frontend/src/components/taskSelection/contributions.js

```diff
@@ -127,10 +127,15 @@ const Contributions = ({ project, tasks, contribsData, activeUser, activeStatus,
   const { percentMapped, percentValidated, percentBadImagery } = useComputeCompleteness(tasks);
 
   let contributionsArray = contribsData.userContributions || [];
+
```

Comment:
@max-keviv could you undo the changes on that file, as we are handling it in another PR?